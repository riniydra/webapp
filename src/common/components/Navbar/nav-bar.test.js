import { render } from "@testing-library/react";
import NavBar from "./NavBar";

describe("NavBar", () => {
  it("renders correctly", () => {
    const tree = render(<NavBar />);
    expect(tree).toMatchSnapshot();
  });
});
