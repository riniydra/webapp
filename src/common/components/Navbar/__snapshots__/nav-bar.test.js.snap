// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar renders correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <header>
        <div
          class="navbar navbar-sm navbar-gradient-bg rounded-3xl px-10"
        >
          <div
            class="navbar-start"
          >
            <mock-link
              props="{"className":"text-3xl font-semibold"}"
              to="/"
            >
              <img
                alt="logo"
                class="w-14 h-14"
                src="logo.svg"
              />
            </mock-link>
          </div>
          <div
            class="navbar-end gap-10"
          >
            <div
              class="dropdown"
            >
              <div
                class="cursor-pointer font-semibold "
                tabindex="0"
              >
                about
              </div>
              <ul
                class="menu menu-md dropdown-content mt-3 z-[1] p-2 shadow bg-base-100 rounded-box w-52 hidden"
                tabindex="0"
              >
                <li>
                  <a>
                    <mock-nav-link
                      props="{}"
                      to="/directors"
                    >
                      directors
                    </mock-nav-link>
                  </a>
                </li>
                <li>
                  <mock-nav-link
                    props="{}"
                    to="/how-we-operate"
                  >
                    how_we_operate
                  </mock-nav-link>
                </li>
                <li>
                  <a>
                    <mock-nav-link
                      props="{}"
                      to="/contact"
                    >
                      contact
                    </mock-nav-link>
                  </a>
                </li>
                <li>
                  <a>
                    <mock-nav-link
                      props="{}"
                      to="/mission-and-vision"
                    >
                      mission
                    </mock-nav-link>
                  </a>
                </li>
                <li>
                  <a>
                    <mock-nav-link
                      props="{}"
                      to="/mission-and-vision"
                    >
                      vision
                    </mock-nav-link>
                  </a>
                </li>
              </ul>
            </div>
            <mock-nav-link
              props="{"className":"font-semibold"}"
              to="/donate"
            >
              donate
            </mock-nav-link>
            <button
              class="font-semibold"
            >
              login
            </button>
          </div>
        </div>
      </header>
    </div>
  </body>,
  "container": <div>
    <header>
      <div
        class="navbar navbar-sm navbar-gradient-bg rounded-3xl px-10"
      >
        <div
          class="navbar-start"
        >
          <mock-link
            props="{"className":"text-3xl font-semibold"}"
            to="/"
          >
            <img
              alt="logo"
              class="w-14 h-14"
              src="logo.svg"
            />
          </mock-link>
        </div>
        <div
          class="navbar-end gap-10"
        >
          <div
            class="dropdown"
          >
            <div
              class="cursor-pointer font-semibold "
              tabindex="0"
            >
              about
            </div>
            <ul
              class="menu menu-md dropdown-content mt-3 z-[1] p-2 shadow bg-base-100 rounded-box w-52 hidden"
              tabindex="0"
            >
              <li>
                <a>
                  <mock-nav-link
                    props="{}"
                    to="/directors"
                  >
                    directors
                  </mock-nav-link>
                </a>
              </li>
              <li>
                <mock-nav-link
                  props="{}"
                  to="/how-we-operate"
                >
                  how_we_operate
                </mock-nav-link>
              </li>
              <li>
                <a>
                  <mock-nav-link
                    props="{}"
                    to="/contact"
                  >
                    contact
                  </mock-nav-link>
                </a>
              </li>
              <li>
                <a>
                  <mock-nav-link
                    props="{}"
                    to="/mission-and-vision"
                  >
                    mission
                  </mock-nav-link>
                </a>
              </li>
              <li>
                <a>
                  <mock-nav-link
                    props="{}"
                    to="/mission-and-vision"
                  >
                    vision
                  </mock-nav-link>
                </a>
              </li>
            </ul>
          </div>
          <mock-nav-link
            props="{"className":"font-semibold"}"
            to="/donate"
          >
            donate
          </mock-nav-link>
          <button
            class="font-semibold"
          >
            login
          </button>
        </div>
      </div>
    </header>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
